client/blocks/edit-gravatar/test/index.jsx

```diff
@@ -118,6 +118,7 @@ describe( 'EditGravatar', function() {
 					<EditGravatar
 						translate={ noop }
 						user={ user }
+						receiveImage={ noop }
```

Comment:
is this something that could be a default prop? if so we wouldn't have to add it to every test